holder/wallet/index.js

```diff
@@ -249,6 +250,8 @@ async function sendCallback(challengeRequest, proof, publicSignals, holderId) {
       // Something happened in setting up the request that triggered an Error
       console.log('Error', error.message);
     }
+    console.log(`Callback to ${url} failed: ${error.message}. Please check the verifier server logs for more details.`)
```

Comment:
If keeping the previous console.log, should this be in an else?
```suggestion
else {
    console.log(`Callback to ${url} failed: ${error.message}. Please check the verifier server logs for more details.`)
}
```

Reply:
This line should be there in a combination with some of the previous logs as they log out more details, but it does mean sometimes it's hard to spot the simple error. 


Probably the last `else` branch is redundant. I'll remove that.

Reply:
done